src/test/scala/io/archivesunleashed/util/TweetUtilsTest.scala

```diff
@@ -73,5 +114,18 @@ class TweetUtilsTest extends FunSuite {
       assert(!tweet.isVerifiedUser())
       assert(tweet.followerCount == expected)
       assert(tweet.friendCount == expected)
+      assert(tweet.retweetCount == expected)
+      assert(tweet.favoriteCount == expected)
+      assert(tweet.inReplyToScreenName() == null)
+      assert(tweet.inReplyToStatusId() == null)
+      assert(tweet.inReplyToUserId() == null)
+      assert(tweet.source() == null)
+      assert(tweet.isUserProtected() == false)
+      assert(tweet.profileImageUrl() == null)
+      assert(tweet.profileDescription() == null)
+      assert(tweet.profileLocation() == null)
+      assert(tweet.profileName() == null)
+      assert(tweet.profileUrl() == null)
+      assert(tweet.profileTimezone() == null)
```

Comment:
Maybe use ".isEmpty" instead of == null, so scalastyle does not burp. Technically the return should be String instead of Null anyway.

Reply:
Just for the assertions. Although this is strictly a style issue, so not mandatory.  (e.g. `assert(tweet.profileLocation().isEmpty)`)

Reply:
Hmm. That weird - these should have been detected and resolved in #249.  I will add the ticket and try to find out why scalastyle did not yell at me.